### PR TITLE
Fixed references to L16.05 for L20.10 build.

### PR DESCRIPTION
**Pull request by rsbeckerca**
This does not represent a full repair. There are issues with NSDEE
configurations relating to different compiler usage that are unreported
as of this commit.

Fixes: #80

Signed-off-by: Randall S. Becker <randall.becker@nexbridge.ca>

### CODE REVIEW

**Review by Jojo-Schmitz — DISMISSED** [review]

**Review by Jojo-Schmitz — DISMISSED** [review]

**Comment by rsbeckerca** [comment]
I think I have this working now. Please review.

File: Makefile.NSE.Win [hunk 1]
```diff
@@ -3,7 +3,8 @@
 VPROC_GMAKE=$(shell sh ./version.sh MAIN)
 
 # Set up environment to use product compiler.
-export COMP_ROOT=C:\Program Files (x86)\HPE NonStop\J06.20
+NONSTOPOSVERSION?=J06.20
```
Comment (Jojo-Schmitz) [inline Makefile.NSE.Win:6]: This is just the default if otherwise unset, right?

File: Makefile.NSX.Win [hunk 2]
```diff
@@ -3,7 +3,8 @@
 VPROC_GMAKE=$(shell sh ./version.sh MAIN)
 
 # Set up environment to use product compiler.
-export COMP_ROOT=C:\Program Files (x86)\HPE NonStop\L16.05
+NONSTOPOSVERSION?=L16.05
```
Comment (Jojo-Schmitz) [inline Makefile.NSX.Win:6]: (same here)

File: Makefile.NSE.Win (hunk 1 above)
Comment (rsbeckerca) [reply Makefile.NSE.Win:6]: Exactly, So passing `NONSTOPOSVERSION=J06.23` on the make line would change COMP_ROOT.

**Review by Jojo-Schmitz — APPROVED** [review]

**Pull request merged**